src/cli/Command.cpp

```diff
@@ -80,14 +80,14 @@ QSharedPointer<QCommandLineParser> Command::getCommandLineParser(const QStringLi
 
     QSharedPointer<QCommandLineParser> parser = QSharedPointer<QCommandLineParser>(new QCommandLineParser());
     parser->setApplicationDescription(description);
-    for (CommandLineArgument positionalArgument : positionalArguments) {
+    for (const CommandLineArgument& positionalArgument : positionalArguments) {
```

Comment:
QString does implicit sharing, but using references is consistent with our coding style.